### ISSUE TITLE
File name integration?

### ISSUE THREAD

**Issue by robotortoise** [TeamFail/3DS-Audio-Dumper#1]
Hi Team_Fail,

Haven't seen you on #zelda or #tcrf lately. Anyways, I saw [this program](https://github.com/andibadra/ANDT/releases) which dumps the file names of sounds as well. I mean, it dumps them as separate text file, but....

 Do you think you could integrate that into your sound file dumper? Yours seems to be the only one available so I'm not sure who else to ask!

**Comment by TeamFail** [comment]
Yeah, I really do need to get back on there some time, I've been really busy with College and a new job. Anyway...

I might be able to add it fairly easily, but the main problem is that if you're dumping from RAM, the file names may not necessarily be available, such as audio samples and whatnot. However, I can make it so that if you're dumping from a BCSAR file I can make it check for names and just use the file pointer table to dump everything instead.

**Comment by robotortoise** [comment]
Ah, glad to hear you're doing well!

But yeah the dumping from a BSCAR thing would be fantastic if you could implement it. I was trying to examine Fire Emblem: Awakening's sound effects the other day, and it has over 2000. It's...a bit much to go through by hand!
